exp-player/addon/components/exp-video-preview/component.js

```diff
@@ -63,7 +62,7 @@ export default ExpFrameBaseComponent.extend(MediaReload, VideoRecord, {
         return this.get('videos')[this.get('videoIndex')];
     }),
     makeTimeEvent(eventName, extra) {
-        return this._super(`exp-physics:${eventName}`, extra);
+        return this._super(`exp-video-preview:${eventName}`, extra);
```

Comment:
Ack. Thanks. 

In the future, we may want to consider standardizing the event name formats: eg omitting the prefix (since it's stored inside a frame payload anyway), or automatically adding the name prefix as part of the base frame logic. 

Within frames, the convention sometimes varies, but I kept them as written to maximize backwards compatibility with existing analysis tools.